Example/src/Example.android.tsx

```diff
@@ -212,71 +215,97 @@ export default class IBeaconExample extends Component {
     startScanning()
       .then(() => this.setState({ scanning: true, statusText: null }))
       .then(() => console.log('started scanning'))
-      .catch(error => console.log('[startScanning]', error));
+      .catch((error) => console.log('[startScanning]', error));
   };
   _stopScanning = () => {
     stopScanning()
-      .then(() => this.setState({ scanning: false, beacons: [], statusText: null }))
+      .then(() =>
+        this.setState({ scanning: false, beacons: [], statusText: null })
+      )
       .then(() => console.log('stopped scanning'))
-      .catch(error => console.log('[stopScanning]', error));
+      .catch((error) => console.log('[stopScanning]', error));
   };
   _restartScanning = () => {
     restartScanning()
-      .then(() => this.setState({ scanning: true, beacons: [], statusText: null }))
+      .then(() =>
+        this.setState({ scanning: true, beacons: [], statusText: null })
+      )
       .then(() => console.log('restarted scanning'))
-      .catch(error => console.log('[restartScanning]', error));
+      .catch((error) => console.log('[restartScanning]', error));
   };
   _isScanning = () => {
     isScanning()
-      .then(result => {
-        this.setState({ statusText: `Device is currently ${result ? '' : 'NOT '}scanning.` });
+      .then((result) => {
+        this.setState({
+          statusText: `Device is currently ${result ? '' : 'NOT '}scanning.`,
+        });
         console.log('Is device scanning?', result);
       })
-      .catch(error => console.log('[isScanning]', error));
+      .catch((error) => console.log('[isScanning]', error));
   };
   _isConnected = () => {
     isConnected()
-      .then(result => {
-        this.setState({ statusText: `Device is ${result ? '' : 'NOT '}ready to scan beacons.` });
+      .then((result) => {
+        this.setState({
+          statusText: `Device is ${result ? '' : 'NOT '}ready to scan beacons.`,
+        });
         console.log('Is device connected?', result);
       })
-      .catch(error => console.log('[isConnected]', error));
+      .catch((error) => console.log('[isConnected]', error));
   };
   _getBeaconRegions = () => {
     getBeaconRegions()
-      .then(regions => console.log('regions', regions))
-      .catch(error => console.log('[getBeaconRegions]', error));
+      .then((regions) => console.log('regions', regions))
+      .catch((error) => console.log('[getBeaconRegions]', error));
   };
 
   /**
    * Helper function used to identify equal beacons
    */
-  _isIdenticalBeacon = (b1, b2) => (
-    (b1.identifier === b2.identifier) &&
-    (b1.uuid === b2.uuid) &&
-    (b1.major === b2.major) &&
-    (b1.minor === b2.minor)
-  );
+  _isIdenticalBeacon = (b1: IBeaconAndroid, b2: IBeaconAndroid) =>
+    b1.major === b2.major && b1.minor === b2.minor;
```

Comment:
Is it safe to stop comparing `identifier` and `uuid` properties?

Reply:
How do you rather want to compare two beacons?